core/services/offchainreporting/OCRContract.go

```diff
@@ -0,0 +1,232 @@
+package offchainreporting
```

Comment:
I think the convention for go filenames is snake case.

Reply:
Yeah, we used PascalCase elsewhere for solidity contract wrappers so I assumed that was some convention.

I don't care too much so happy to change it to snek_case if that's less surprising